system_tests/datastore.py

```diff
@@ -271,7 +287,13 @@ def test_projection_query(self):
         self.assertEqual(catelyn_stark_dict,
                          {'name': 'Catelyn', 'family': 'Stark'})
 
-        catelyn_tully_entity = entities[3]
+        if EMULATOR_DATASET is None:
```

Comment:
w00t! Huge :+1: for making the emulator essentially trivial to work with.

Do you advise using our indices for the emulator? As can be seen in #1370, I am just populating the stark family tree at test case set-up and then deleting them on completion.